vault/dendron.release.2021-11-09.md

```diff
@@ -0,0 +1,92 @@
+
+* Faster webview performance
+* Clipboard support (copy and paste works)!
+* Live color theme updates (vs reloading your window for changes to take effect)
+* Bundle native preview with webpack
+* Build native preview on `yarn setup`
+* Cleanup of assets 
+
+### Highlights
+* feat(schemas): adding new command - create schema from hierarchy ([[docs|dendron.topic.schema.create-from-hierarchy]])
+* feat(views): native dendron preview
+* feat(publish): add local table of contents, of current note, in right-hand nav ([[docs|dendron.topic.publish.features#table-of-contents]])
+* feat(publish): merge lookup and search fields ([[docs|dendron.topic.publish.features#lookup-and-search]])
+
+### Everything Else
+* enhance(schemas): adding error messaging and additional validation for when schemas are malformed
+* enhance(schemas): cmd create-schema-from-hierarchy allow schema to end with `*`
+* enhance(schema): update schemas on save instead of having to reload workspace
+* enhance(commands): move header supports multi vault
+* enhance(commands): allow creation of new notes when move header destination doesn't exist yet
+* fix(pods): github import pod handle deleted authors
+* fix(workspace): warning to remove stub property if the note has content
+* fix(publish): table of contents layout
+* fix(server): improving error response on api server
+* fix(workspace): Change Workspace command recognizes native workspaces
+
+#### Breaking Changes
+* enhance(publish): migrate publish related commands to nextjs @hikchoi
+
+Transitioning site build and previewing commands to use nextjs (instead of 11ty), along with the renaming of commands to be more inline with CLI:
+* `Dendron: Site Preview` is renamed to `Dendron: Publish Dev`
+* `Dendron: Site Build` is renamed to `Dendron: Publish Export`
+
+### Community
+
+#### General Updates
+
+- Join the discussion around the [Dendron Site Reorganization](https://github.com/dendronhq/dendron/discussions/1665), where we're refactoring the hierarchies for a better user experience
+- We are making some changes to our [newsletter](https://buttondown.email/dendron) (heads up, we have a newsletter). We are changing it from a monthly (lets be honest, once every few months) newsletter to a weekly one. The newsletter will feature release highlights, community highlights, and more! For more details, see the [[community.email-newsletter]].
+- New Discord channel, `#today-i-learned`, is a great place to share discoveries about Dendron and anything else
+
+#### Google Docs Import Pod: Sign up for the beta
+
+We're rolling out [[direct integration with google docs|dendron.topic.pod.builtin.google-docs.import]] from inside of dendron. we have used this extensively, internally, and are now looking for external beta testers before rolling it out generally. If interested, please [fill out the survey](https://airtable.com/shrP1yKjIDPFU4wHN) to get access to the beta!
```

Comment:
add reason why survey is needed (aka - to use the pod, we'll need to whitelist your email with our google app api)